General expression summaries (#291)
Certora/Documentation: docs/cvl/methods.md

 - {ref}`dispatcher` assume that the receiver of the method call could be any
   contract that implements the method.

>  - {ref}`expression-summary` replace calls to the summarized method with a CVL expression, typically {doc}`functions`

Reviewer: ```suggestion
 - {ref}`function-summary` replace calls to the summarized method with a CVL expression, typically {doc}`functions`
```

  [State Mutability](https://docs.soliditylang.org/en/v0.8.12/contracts.html#state-mutability)
  in the Solidity manual for details.

> (expression-summary)=
> #### Expression summaries

Reviewer: I'd leave the change minimal.
```suggestion
(function-summary)=
#### Function and Expression summaries
```

I think we want to leave the focus that `CVL functions` are used as summaries. From a spec writing perspective this is the cleaner solution.

Reviewer 2: I'm actually fine with changing the refname as well.

> Contract methods can also be summarized using CVL expressions, typically {doc}`functions` or

Reviewer: Maybe?

```suggestion
Contract methods can also be summarized using CVL functions or expressions (for instance {ref}`ghost-axioms` as approximations).
```
